packages/components/src/components/accordion/accordion-styles.ts

```diff
@@ -115,6 +121,15 @@ export const getComponentCss = (
     },
     heading: {
       margin: 0,
+      ...(sticky && {
+        position: 'sticky',
+        top: 0,
+        zIndex: zIndexes.indexOf('heading'),
```

Comment:
why not simply define `zIndex: 1` (maybe with a comment) then we'd save unnecessary bytes in the bundled chunk?

Reply:
Coming from a Sass background I found it way more intuitive to define such z-index ordering as a list, but you are right, we need to primarily safe bytes and a comment should be fine.